Assets/Engine/Interactions/IntentManager.cs

```diff
@@ -19,13 +19,27 @@ public class IntentManager : MonoBehaviour
     public Color colorHarm;
     public Color colorHelp;
 
+    private Button IntentButton;
+
     public enum IntentType
     {
         Help,
         Harm
     }
+
+    void Start()
+    {
+        IntentButton = GetComponent<Button>();
+        IntentButton.onClick.AddListener(TaskOnClick);
+    }
+
+    void TaskOnClick()
```

Comment:
Access level modifiers should be explicitly defined for classes, methods and member variables. This includes defining private even though C# will implicitly add it.